lesson_11/roman_alex/lesson_11_roman_alex.py

```diff
@@ -0,0 +1,59 @@
+from collections.abc import MutableMapping, Mapping, Iterable, Collection, Container, Sized
+
+
+class Student(object):
+
+    def __init__(self, first_name, last_name, age):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.age = age
+
+    def __repr__(self):
+        class_name = type(self).__name__
+        return '{} ({} {} - {}) [{}]'.format(
+            class_name, self.first_name,
+            self.last_name, self.age, id(self)
+        )
+
+    def __str__(self):
+        return '{} {}, {}'.format(
+            self.first_name, self.last_name, self.age)
+
```

Comment:
Hello, you will also need to add a __len__ method ( for the sized property)

Reply:
Code updated and len method added